scripts/test_examples.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -e
+
+# List files changed in the commit to check
+FILES=`git log -n 2 --name-only --format=""`
```

Comment:
@tengqm do you need the last 2 commits or only latest one ?

Reply:
Good question. It has to be 2 because according to the travis log, the following operations are automatically performed during environment preparation:

```
git clone --depth=50 https://github.com/kubernetes/website.git kubernetes/website
cd kubernetes/website
git fetch origin +refs/pull/<PR number>/merge:
git checkout -qf FETCH_HEAD
```

After the above operations, the latest two commits to website repo are the 'merge' operation and the actual changes made by your PR. The "`-n 2`" was carefully chosen to ensure we are examining *exactly* the set of files touched by the PR under testing.

Reply:
ah right - missed that info, thank you for the details @tengqm .